lib/state/action-types.ts

```diff
@@ -93,6 +94,7 @@ export type ActionType =
   | Search
   | SelectNote
   | SelectRevision
+  | SelectTag
```

Comment:
Let's match names here with the action type? Or do you see value in keeping this `SelectTag`?